chart/templates/secret.yaml

```diff
@@ -15,7 +15,11 @@ kind: Secret
 metadata:
   name: "bootstrap-secret"
   namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "-5"
```

Comment:
Any particular reason why you set the weight to -5 rather than the default 0? Not arguing it should be 0, I'm just curious about the reasoning since I haven't used hook weights before.

Reply:
It's taken from the docs https://helm.sh/docs/topics/charts_hooks/, I left it to be sure it was low enough I guess but I don't think it really matters